x-pack/plugins/aiops/public/components/change_point_detection/change_points_table.tsx

```diff
@@ -116,7 +116,6 @@ export const ChangePointsTable: FC<ChangePointsTableProps> = ({
       height: '80px',
       truncateText: false,
       valign: 'middle',
-      css: { display: 'block', padding: 0 },
```

Comment:
Is there anything we can do about retaining the `0` padding here? As it is, the height of the rows increases with more unnecessary whitespace. 

Before:
<img width="269" alt="change_point_progress1" src="https://github.com/elastic/kibana/assets/7405507/66bf6032-2a1b-4e55-8a7c-79b5067de217">

After:
<img width="460" alt="change_point_progress2" src="https://github.com/elastic/kibana/assets/7405507/8fd4e93b-f282-4691-aadb-7587fc48abbe">

Reply:
🤦 Thanks so much for catching that Pete, I see what was happening now. I've restored the original CSS with an extra specificity selector so that the Emotion CSS overrides EUI's not-yet-converted Sass CSS: https://github.com/elastic/kibana/pull/161592/commits/60c382a4b70bf88de9be752ad9bf8647842a2e9a

The TODO is for myself/EUI to remember to remove once EuiTable is converted to Emotion.